hera_qm/tests/test_metrics_io.py

```diff
@@ -465,11 +465,12 @@ def test_process_ex_ants_bad_string():
 
 def test_process_ex_ants_string_and_file():
     ex_ants = '0,1'
-    met_file = os.path.join(DATA_PATH, 'example_ant_metrics.hdf5')
-    xants = metrics_io.process_ex_ants(metrics_files=met_file)
-    assert xants == [81]
-    xants = metrics_io.process_ex_ants(ex_ants=ex_ants, metrics_files=met_file)
-    assert xants == [0, 1, 81]
+    for ext in ['hdf5', 'json']:
```

Comment:
Instead of doing this as a `for` loop inside of the test, we could pretty easily make this into two separate tests using [pytest's parametrize functionality](https://docs.pytest.org/en/6.2.x/parametrize.html), which is relatively easy to implement. I think that's better testing practice because if one test fails but not the other, it makes it more obvious where the problem is coming from.

Reply:
Good idea! Done.